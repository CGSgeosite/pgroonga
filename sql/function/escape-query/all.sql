SELECT pgroonga.escape_query('a+B-c< >あいう~*()"\\'':');
