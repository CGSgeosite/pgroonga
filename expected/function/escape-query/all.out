SELECT pgroonga.escape_query('a+B-c< >あいう~*()"\\'':');
            escape_query             
-------------------------------------
 a\+B\-c\< \>あいう\~\*\(\)\"\\\\'\:
(1 row)

